cmd/ingester/app/consumer/consumer_metrics.go

```diff
@@ -36,6 +37,7 @@ func (c *Consumer) newMsgMetrics(partition int32) msgMetrics {
 		counter:     f.Counter("messages", nil),
 		offsetGauge: f.Gauge("current-offset", nil),
 		lagGauge:    f.Gauge("offset-lag", nil),
+		lagGaugeNs:  f.Gauge("offset-lag-ns", nil),
```

Comment:
`-ns` is implementation specific. Need a different name. Also, "offset" isn't applicable for this measure. I think simply "lag" will work.

Reply:
I agree with what you say with the name of the metric, `lag` works. I agree that `-ns` is implementation specific, perhaps this is a reason to use a `Timer` because `Timer` metrics have a notion of measuring in units of time, as opposed to gauges which may refer to anything.